spec/unit/transaction/persistence_spec.rb

```diff
@@ -148,7 +148,7 @@ def write_state_file(contents)
       persistence = Puppet::Transaction::Persistence.new
 
       if Puppet.features.microsoft_windows?
-        expect { persistence.save }.to raise_error(Puppet::Util::Windows::Error, /Access is denied/)
+        expect { persistence.save }.to raise_error(Puppet::Util::Windows::Error)
```

Comment:
Actually, even if it did, there's no way to express that in the raise_error matcher.  We're restricted to regex on strings

Example message;
```
Puppet::Util::Windows::Error: ReplaceFile(C:/Users/Administrator/AppData/Local/Temp/rspecrun20161104-3864-ktdcn5/persistence_test20161104-3864-60f4wh, C:/Users/Administrator/AppData/Local/Temp/rspecrun20161104-3864-ktdcn5/persistence_test20161104-3864-60f4wh20161104-3864-vl57cp):  Accès refusé.
from C:/Source/puppet/lib/puppet/util/windows/file.rb:67:in `replace_file'
```

https://www.relishapp.com/rspec/rspec-expectations/v/3-5/docs/built-in-matchers/raise-error-matcher.